RELEASE.md

```diff
@@ -226,6 +226,10 @@ git checkout release-<version>
 > **Note:** Technical documentation is automatically published on release tags or release branches with a corresponding
 > release tag. The workflow that publishes documentation is defined in [`publish-technical-documentation-release.yml`](.github/workflows/publish-technical-documentation-release.yml).
 
+> **Note:** Homebrew's [autobump job](https://github.com/Homebrew/homebrew-core/actions/workflows/autobump.yml)
+> automatically updates the [`mimirtool` Homebrew formula](https://github.com/Homebrew/homebrew-core/blob/master/Formula/mimirtool.rb).
+> The autobump job runs once every 24 hours, so it may take a day or two for the updated formula to be created and published.
```

Comment:
```suggestion
> The autobump job runs once every 24 hours, so it might take a day or two for the updated formula to be created and published.
```
Rationale: Consistency, and because may generally connotes permission, and might generally connotes possibility.